src/main/java/eod/warObject/character/concrete/red/BloodThirstFighter.java

```diff
@@ -0,0 +1,99 @@
+package eod.warObject.character.concrete.red;
+
+import eod.GameObject;
+import eod.Party;
+import eod.Player;
+import eod.card.abstraction.summon.SummonCard;
+import eod.card.concrete.summon.BloodThirstFighterSummon;
+import eod.effect.EffectExecutor;
+import eod.effect.RegionalAttack;
+import eod.event.Event;
+import eod.event.ObjectEnterEvent;
+import eod.event.relay.EventReceiver;
+import eod.param.PointParam;
+import eod.warObject.Damageable;
+import eod.warObject.Status;
+import eod.warObject.character.abstraction.assaulter.Fighter;
+
+import java.awt.*;
+import java.util.ArrayList;
+
+import static eod.effect.EffectFunctions.RequestRegionalAttack;
+
+public class BloodThirstFighter extends Fighter {
+    public BloodThirstFighter(Player player) {
+        super(player, 4, 3, Party.RED);
+        new OwnedAbilities(this);
+    }
+
+    @Override
+    public SummonCard getSummonCard() {
+        SummonCard c = new BloodThirstFighterSummon();
+        c.setPlayer(player);
+        return c;
+    }
+
+    @Override
+    public String getName() {
+        return "嗜血的戰鬥狂";
+    }
+
+    @Override
+    public void attack(EffectExecutor executor) {
+        super.attack(executor);
```

Comment:
Same as [this](https://github.com/Elegy-Composer/EOD-DM/pull/35/files#r430118507)

Reply:
fixed